benchmarks/nvector/kokkos/test_nvector_performance_kokkos.cpp

```diff
@@ -228,15 +228,15 @@ void sync_device(N_Vector x)
  * Functions required for clearing cache
  * --------------------------------------------------------------------*/
 
-int InitializeClearCache(SizeType cachesize)
+int InitializeClearCache(sunindextype cachesize)
```

Comment:
@gardner48  I don think this should be a `sunindextype` since it has no relation to indexing or sundials core. `size_t` would be more appropriate I think.

Reply:
Agreed `size_t` would be best here.